devito/finite_differences/derivative.py

```diff
@@ -312,6 +312,14 @@ def evaluate(self):
         # Evaluate finite-difference.
         # Note: evaluate and _eval_fd splitted for potential future different
         # types of discretizations.
+        do = self.deriv_order
```

Comment:
Although `do` is not used in python (I think?), I would avoid using it as a param name for obvious reason of naming converntions, maybe keep deriv_order ?

Reply:
Conceptually, it is still a space order though right ? (deriv_so ?)

Reply:
@georgebisbas, we have already discarded this approach (PR still a draft). It doesn't really solve the problem (see my last comment responding Fabio's comment).  Anyway thanks for the comment. I agree with you.